manage.py

```diff
@@ -475,6 +475,13 @@ def load_efile_sheets():
         df.drop(columns_to_drop, axis=1, inplace=True)
         df.to_json(path_or_buf="data/" + table + ".json", orient='values')
 
+@manager.command
+def refresh_calendar():
+    """ Refreshes calendar data
+    """
+    with db.engine.begin() as connection:
+        connection.execute('REFRESH MATERIALIZED VIEW CONCURRENTLY ofec_omnibus_dates_mv')
```

Comment:
This will refresh without issue, right?  Meaning there are no dependencies we need to be aware of?